### PR TITLE
Downloading playlists feature

### PR DESCRIPTION
**Pull request by vishesh98**

### CODE REVIEW

**Comment by Parth-Vader** [comment]
Great work, but there are some problems with it:

* It prints an empty space while downloading several files of a playlist.
*  I think a commit is missing since [this line](https://github.com/Parth-Vader/MobOff/blob/master/moboff.py#L157) has been replaced by `The file has been sent to your phone.`
* It creates a folder called `files_go_here` inside the specified download folder, while downloading.

**Comment by vishesh98** [comment]
files_go_here is temporarily created.It's deleted at the end after shifting the files to selected directory.
Should I add something like  "this may take a while " for when playlists are being downloaded?

**Comment by Parth-Vader** [comment]
It would be better if the folder isn't created. The other two points are also important to address, and yes it would be better to write this may take a while instead.

**Comment by vishesh98** [comment]
If all the files are stored in the same location( the selected directory), then pushing just the recently downloaded files might be complex as you dont want to push the all the files( even those that were downloaded earlier but not deleted ) every time. Also if the user while downloading a playlist downloads another file in the same folder we dont want to push that. So I thought of creating a temporary folder which will have  the files that were just downloaded using moboff. It makes it easier to push the and then they are shifted to the required directory and the temporary folder is deleted.

I couldn't think of a way to do this without a temp. folder.

File: moboff.py [hunk 1]
```diff
@@ -150,7 +152,7 @@ def download(link, newdevice, video, delete,send):
         with open(file, "rb") as song:
             file_data = pb.upload_file(song, file)
         to_device.push_file(**file_data)
-        print("The file has been  sent your phone.")
+        print("The file has been to sent your {0}.".format(to_device))
```
Comment (Parth-Vader) [inline moboff.py]: The sentence is wrong.

**Pull request merged**